db/schema.rb

```diff
@@ -220,6 +220,7 @@
     t.jsonb "accrediting_provider_enrichments"
     t.float "latitude"
     t.float "longitude"
+    t.string "provider_name_search", null: false
```

Comment:
since this is `not null` why is its dependent provider_name `nullable`?

Reply:
Good question, I just assumed the provider_name would not be nullable